test/functional/task.spec.js

```diff
@@ -3,6 +3,7 @@
 const { test, trait} = use('Test/Suite')('Task')
 const Task = use('App/Models/Task')
 trait('Test/ApiClient')
+//add some changes here
```

Comment:
good job 😄 !!

To have a more readable unit test please follow this : 

https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80

Reply:
Okey!